scripts/build.js

```diff
@@ -70,7 +70,7 @@ const args = parseArgs(process.argv.slice(2), {
       const { host, port } = await ctx.serve(
         {
           servedir,
-          host: 'localhost',
+          host: '0.0.0.0',
```

Comment:
Why we change this?

Reply:
I needed to test playground in iOS simulator to see their behaviour in iOS, and since its `localhost` is different than my system localhost, I couldn't reach it. `0.0.0.0` means listening local IP, so I can reach playground with the IP of my computer.

After that, I used `ngrok` to test it though. Maybe I can revert this back.

Reply:
With localhost there, you can try 127.0.0.1. It should work.

Reply:
Yeah, maybe it works that way too. Either way, reverted this change back. 👍🏻

Reply:
I mean you can try 127.0.0.1 in the simulator with our default configurations as well. So either way reverting back is better for me too.